### PR TITLE
fix(ComponentDemo): add titleText prop to Dropdowns

### LINKED ISSUE #1550: audit-avt1-accordion: down-shift label
## Environment:

> Operating system: Mac
> Browser: Chrome 
> AVT :one:
> Page: [Components - Accordion: Style](https://www.carbondesignsystem.com/components/accordion/code)

## Detailed Description: 

The 'id' "downshift-5-label" specified for the WAI-ARIA property 'aria-labelledby' value is not valid

Screen Shot 2020-08-11 at 1 24 49 PM

## What to do: 
Update the WAI-ARIA property value with the unique id value of a valid visible element.

## Why is this important?
Any element that uses a WAI-ARIA property must reference a valid id attribute. This enables assistive technologies to precisely identify the referenced element.

### PR DESCRIPTION
**Pull request by dakahn**
Closes #1550 

Accessibility Checker was throwing errors for our unlabeled Dropdowns in the Component Demo. This PR adds `titleText` props so the elements are clearly labeled.

image

### Notes

Text alignment seems a little wonky with the new label text on the page @carbon-design-system/design might want to advise.

### CODE REVIEW

**Comment by dakahn** [comment]
@andreancardona was looking at the preview and found a bug
image

Looks like we're rendering `titleText` whether the Dropdown is visible or not 🤦

File: src/components/ComponentDemo/ComponentDemo.js [hunk 1]
```diff
@@ -102,6 +102,7 @@ const ComponentDemo = ({ children, src, scope, noInline, components }) => {
             initialSelectedItem="White"
             id="theme-variant"
             label="Theme variant selection"
+            titleText="Theme variant selection"
```
Comment (andreancardona) [inline src/components/ComponentDemo/ComponentDemo.js:105]: `  titleText={
              childrenArray.length === 1 ? '' : 'Component variant selection'
            }`

Comment (andreancardona) [inline src/components/ComponentDemo/ComponentDemo.js:105]:
```suggestion
            titleText={ childrenArray.length === 1 ? '' : 'Component variant selection' }
```

Comment (andreancardona) [reply src/components/ComponentDemo/ComponentDemo.js]: @dakahn just tested this and it seems to work :)

**Pull request merged**